environment-idr-omero53.yml

```diff
@@ -0,0 +1,16 @@
+# conda env create -f environment-idr-omero53.yml
+name: idr-omero53
```

Comment:
As above, either a name here or on the command-line. (My preference would be none in the file).